python/ycm/vimsupport.py

```diff
@@ -21,6 +21,7 @@
 import os
 import tempfile
 import json
+import re
```

Comment:
Forgive me if I missed it, but is this new import used? I can't see a reference to re in the diff.